_members/megan_kraus.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Megan Kraus
+image: 
```

Comment:
Edit to: 
`image: images/team/megan_kraus.jpg`

Reply:
Complete